foundry/collections/entityCollections/users.d.ts

```diff
@@ -22,6 +22,11 @@ declare class Users extends EntityCollection<User> {
    */
   get players(): User[];
 
+  /**
+   * An object of key-value permissions for the Users.
+   */
+  static permissions: any;
```

Comment:
This should probably be at the very bottom of the class to reflect where it is set in `foundry.js`. Also The type should probably be `Const.UserPermission` (assuming that this type already exists – if it doesn't, it should be created).

Also we usually don't introduce our own documentation, unless there is a pressing reason to do so. In this case, `permissions` is not documented in `foundry.js`, so it shouldn't be documented here.